templates/zerver/integrations.html

```diff
@@ -969,6 +969,33 @@
 
     </div>
 
+    <div id="gosquared" class="integration-instructions">
```

Comment:
You gave possibility to pass custom topic by a user  (by passing it to url/request I think)
so you should explain the concept in the doc.